cmd/bots/internal/common/templates/alerts/base_target_alert.html

```diff
@@ -3,10 +3,9 @@
 <b>Level:</b> Error ❌
 
 <b>Details:</b> Base target is greater than the threshold value. The threshold value is {{ .Threshold}}
-
 {{ with .BaseTargetValues }}
 {{range .}}
-    Node: <code>{{ .Node}}</code>
-    Base Target: <code>{{ .BaseTarget}}</code>
+Node: <code>{{ .Node}}</code>
```

Comment:
Looks like one space is missed before closing double curly braces `}}`. And below.

Reply:
I know that it's just formatting.